File: app/src/main/java/com/fieldbook/tracker/brapi/Trait.java
package com.fieldbook.tracker.brapi;

import java.util.List;

public class Trait {

    private String name;
    private String description;
    private String traitDbId;

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }


    public void setTraitDbId(String traitDbId) {
        this.traitDbId = traitDbId;
    }

    public String getTraitDbId() {
        return traitDbId;
    }

}

File: app/src/main/java/com/fieldbook/tracker/cropontology/models/Trait.kt
package com.fieldbook.tracker.cropontology.models

import com.google.gson.annotations.SerializedName

data class Trait(
    val traitDbId: String?,
    @SerializedName("name")
    val traitName: String?,
    @SerializedName("class")
    val traitClass: String?,
    @SerializedName("description")
    val traitDescription: String?,
    val entity: String?,
    val attribute: String?,
) {
    override fun toString(): String {
        return "\n\t Name: $traitName \n\t Class: $traitClass \n\t $entity \n\t $attribute \n\t $traitDescription"
    }
}